server/views/applications/start.njk

```diff
@@ -33,15 +33,14 @@
         <ul>
           <li>to have consent from the person on probation to refer for TA</li>
           <li>to have completed a Duty to Refer and Commissioned Rehabilitative Services referral</li>
-          <li>a signed <a href="">Accommodation Compact</a> from the person</li>
           <li>the person's CRN</li>
           <li>an up to date OASys – we'll use this to pre-fill some of your referral</li>
         </ul>
 
         <h3>You need to know</h3>
 
         <ul>
-          <li><a href="">eligibility requirements</a> for TA. You'll need to confirm that all other accommodation options have been considered</li>
+          <li>eligibility requirements for TA (outlined within the <a href="https://equip-portal.equip.service.justice.gov.uk/CtrlWebIsapi.dll/?__id=webDiagram.show&map=0%3AFF2D8D3F16B44268B814F7F8177A16F7&dgm=6B8C1C46E26D4ACFBEC3D16F7ABA3029">Operational Guidance for TA</a>)</li>
```

Comment:
We repeat "for TA" so we might be able to drop the second in favour of just "Operational Guidance" but it really doesn't matter much!